include/fc/bloom_filter.hpp

```diff
@@ -11,16 +9,20 @@
  *                                                                   *
  * Copyright notice:                                                 *
  * Free use of the Open Bloom Filter Library is permitted under the  *
- * guidelines and in accordance with the most current version of the *
- * Common Public License.                                            *
- * http://www.opensource.org/licenses/cpl1.0.php                     *
+ * guidelines and in accordance with the MIT License.                *
+ * http://www.opensource.org/licenses/MIT                            *
  *                                                                   *
  *********************************************************************
 */
 
+
+#ifndef INCLUDE_BLOOM_FILTER_HPP
+#define INCLUDE_BLOOM_FILTER_HPP
```

Comment:
Why the removal of #pragma once? I do not believe any compiler within our target platforms have a problem with it. IMHO these are a bit ugly.

Reply:
Because upstream does it like this.